addons/event/views/event_views.xml

```diff
@@ -9,116 +9,63 @@
+                               </div>
+                               <div colspan="2" class="o_checkbox_optional_field">
+                                   <label for="tag_ids"/>
+                                   <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'no_quick_create': True}"/>
+                               </div>
+                           </group>
+                           <group>
+                               <div colspan="2" class="o_checkbox_optional_field">
+                                   <label for="has_seats_limitation" string="Limit Registrations"/>
+                                   <field name="has_seats_limitation"/>
+                                   <span attrs="{'invisible': [('has_seats_limitation', '=', False)], 'required': [('has_seats_limitation', '=', False)]}">
+                                       to <field name="seats_max" class="oe_inline"/>
+                                       Attendees
+                                   </span>
+                               </div>
+                               <div colspan="2" class="o_checkbox_optional_field">
+                                   <label for="auto_confirm" string="Autoconfirmation"/>
+                                   <field name="auto_confirm"/>
+                               </div>
+                           </group>
+                       </group>
+
+                       <notebook>
+                           <page string="Tickets">
+                               <field name="event_type_ticket_ids"
+                                      class="w-100"
+                                      context="{
+                                         'tree_view_ref': 'event.event_type_ticket_view_tree_from_type',
+                                         'form_view_ref': 'event.event_type_ticket_view_form_from_type'
+                                      }"
+                               />
+                           </page>
+                           <page string="Communication" name="event_type_communication">
+                               <field name="event_type_mail_ids" class="w-100">
+                                   <tree string="Communication" editable="bottom">
+                                       <field name="notification_type" invisible="1"/>
+                                       <field name="template_id" attrs="{'required': [('notification_type', '=', 'mail')]}"/>
+                                       <field name="interval_nbr" attrs="{'readonly':[('interval_unit', '=', 'now')]}"/>
+                                       <field name="interval_unit"/>
+                                       <field name="interval_type"/>
+                                   </tree>
+                               </field>
+                           </page>
+                           <page string="Notes" name="event_type_notes">
```

Comment:
Why a name ? Unless necessary for override (which does not seem to be the case) we generally don't add it.

Reply:
Done, name removed.